.github/workflows/release.yaml

```diff
@@ -67,4 +71,16 @@ jobs:
         env:
           IMAGE_URI_DIGEST: ${{ env.IMAGE_URI }}@${{ steps.build_n_publish_image.outputs.IMAGE_DIGEST }}
           GITHUB_ACITONS_OIDC_ISSUER: https://token.actions.githubusercontent.com
-          COSIGN_KEYLESS_SIGNING_CERT_SUBJECT: https://github.com/${{ github.repository }}/.github/workflows/release.yaml@${{ github.ref }}
\ No newline at end of file
+          COSIGN_KEYLESS_SIGNING_CERT_SUBJECT: https://github.com/${{ github.repository }}/.github/workflows/release.yaml@${{ github.ref }}
+
+  provenance-container:
+    name: generate provenance for container
+    needs: [build-n-release]
+    if: startsWith(github.ref, 'refs/tags/')
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@v1.9.0 # must use semver here
```

Comment:
do we want to use the same version for the slsa generator in the guacsec org? In that case we should update the guac repo to use the new version too? https://github.com/guacsec/guac/blob/v0.1.2/.github/workflows/release.yaml#L137C93-L137C123

Reply:
PR to be created in the guac repo to upgrade the versions there.